src/app/data_harvester/memory/sysinfo.rs

```diff
@@ -36,3 +36,21 @@ pub(crate) fn get_swap_usage(sys: &System) -> Option<MemHarvest> {
         },
     })
 }
+
+/// Returns cache usage. sysinfo has no way to do this directly but it should equal the difference
+/// between the available and free memory. On windows, this will always be 0.
+#[cfg(not(target_os = "windows"))]
+pub(crate) fn get_cache_usage(sys: &System) -> Option<MemHarvest> {
+    let mem_used_in_kib = (sys.available_memory() - sys.free_memory()) / 1024;
+    let mem_total_in_kib = sys.total_memory() / 1024;
+
+    Some(MemHarvest {
+        total_kib: mem_total_in_kib,
+        used_kib: mem_used_in_kib,
+        use_percent: if mem_total_in_kib == 0 {
+            None
+        } else {
+            Some(mem_used_in_kib as f64 / mem_total_in_kib as f64 * 100.0)
+        },
+    })
+}
```

Comment:
Hmmm, is there any documentation for this? Just double checking since if I load up `htop` on Linux, it reports cache as 17.1 GiB, while this is reporting 18.7 GiB, which matches `free`. Might be good to include any documentation on the calculation here in the comment.